### PR TITLE
directory lazy-field for project

### PR DESCRIPTION
**Pull request by aaronkchsu**

### CODE REVIEW

**Comment by CarsonF** [comment]
Hey @aaronkchsu we actually want `rootDirectory` to return a `Directory` object. We can fetch the children from that and graphql will handle that nested call.

**Comment by aaronkchsu** [comment]
So we will only return Directory object in this call? 
Then I am not sure about the response structure.
rootDirectory {
     files {
          items {
          }
     }
}

**Comment by CarsonF** [comment]
Yep just return a Directory from the resolver. That is the response structure.

File: src/components/file/dto/list.ts [hunk 1]
```diff
@@ -25,6 +25,12 @@ export abstract class FileFilters {
     nullable: true,
   })
   readonly category?: readonly FileNodeCategory[];
+
+  @Field(() => ID, {
+    description: 'Only childrens in root directory of this project',
+    nullable: true,
+  })
+  readonly projectId?: string;
```
Comment (CarsonF) [inline src/components/file/dto/list.ts:33]: This isn't needed

**Pull request merged**